website/static/css/custom.css

```diff
@@ -72,6 +72,31 @@
     box-sizing: unset;
 }
 
+/* Fix the klaro */
+.klaro .cookie-modal .cm-modal .hide {
+    top: 20px !important;
+    display: inline-block;
+}
+
+.cm-toggle-all {
+    display: none !important;
```

Comment:
What is `.cm-toggle-all`? I didn't notice anything changine when I removed this rule. And is `display: none;` the best way to disable it? Maybe it is, but red flags go off when I see `!important`. (Same applies to the `top` rule above.

I found this article helpful about `!important` https://css-tricks.com/when-using-important-is-the-right-choice/

ButI actually didn't notice these css rules doing anything.

Reply:
`cm-toggle-all` is the element which allows you to toggle all "klaro apps" at once:

<img width="752" alt="image" src="https://user-images.githubusercontent.com/1860335/64465798-ca94af80-d10e-11e9-910c-74c86c7ca57c.png">


We only have one choice, so I agree, it makes sense to hide it, unless there is a specific configuration we can set rather than hiding it with CSS